samples/voting/README.md

```diff
@@ -9,11 +9,44 @@ The project should be immediately runnable by calling `tye run` from the directo
 
 A few things need to be configured before deploying to Kubernetes.
 
-- Setting up Redis. A connection string needs to be provided to connect to Redis. You can follow our tutorial on [setting up redis in your cluster](../../docs/tutorials/hello-tye/02_add_redis.md).
-- Setting up postgresql. A connection string eventually needs to be provided to tye for postgresql.
-- Deploying the ingress.yaml by calling:
+- [Create an Azure container registry using the Azure portal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal) or you can use your public [docker hub account](https://hub.docker.com/)
+
+- `Redis` can be deployed using the below command :
 
     ```
-    kubectl apply -f ingress.yml
+    kubectl apply -f https://raw.githubusercontent.com/dotnet/tye/master/docs/tutorials/hello-tye/redis.yaml
+    ```
+- `Postgresql` can be installed by the following command :
```

Comment:
Can you also mention that you can install postgres and redis with an external service, like a managed redis instance?

Reply:
Rephrased the `redis` and `postgres` installation steps to include the information related to managed service.

Reply:
Updated this step already.